packages/celotool/src/cmds/deploy/migrate/blockscout.ts

```diff
@@ -1,7 +1,6 @@
-import { getInstanceName, getReleaseName, installHelmChart } from 'src/lib/blockscout'
+import { getReleaseName, installHelmChart } from 'src/lib/blockscout'
```

Comment:
What is the function of `migrate` command? Is it for installing a "new" blockscout release when the secret already exits?

Reply:
It's hard to say, we've never used it really.